CONTRIBUTE.md

```diff
@@ -0,0 +1,56 @@
+
+# Run the demo environment 
+
+# Step 1: Setup and Compile
+
+* Compile the yard repository with **mvn clean install**
+    * https://github.com/kiegroup/yard
+* Compile this project **mvn clean install**
+* In order to run the Backstage plugin the repository with the Akrivis plugin is required
+  * Current Backstage plugins repository 
+    * https://github.com/kiegroup/backstage-plugins
+    * Branch name: **akrivis**
+  * It might be necessary to run **yarn install** in Backstage plugins repository
```

Comment:
i am not sure if this shouldn't be in more required form?

it is true that if you compile it for the first time it is required, right? but maybe even if yarn.lock file is not up to date with branch updates we still need to run yarn anyway.

Reply:
Since the yarn.lock is committed with the latests changes, is should always be up to date. However we could just make this requirement for this document to make it more easier to read and implement.